src/server/constants.js

```diff
@@ -13,3 +13,5 @@ exports.LockerStates = {
 	RESERVED: 3,
 	RENTED: 4,
 };
+
+exports.MaxRentals = 5;
```

Comment:
This should be all caps, to follow the usual convention for constants.

For tracking purposes: #42

Reply:
Done!